src/event_manager.h

```diff
@@ -24,36 +24,34 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <list>
 #include <map>
 
-class EventManager: public MtEventManager
+class EventManager : public MtEventManager
 {
 	static void receiverReceive(MtEvent *e, void *data)
 	{
-		MtEventReceiver *r = (MtEventReceiver*)data;
+		MtEventReceiver *r = (MtEventReceiver *)data;
```

Comment:
because the rule for pointers is to have * at the right, then casts are affected